conf/livy-defaults.conf

```diff
@@ -0,0 +1,43 @@
+# Specifies Livy's environment. May either be "production" or "development". In "development"
```

Comment:
Could you remove this file? It already existed with a `.template` extension, and it's now called `livy.conf`.